### ISSUE TITLE
Tips for enhancing the performance of Fast RCNN w/ Alex-net

### ISSUE THREAD

**Issue by Changjae** [nightrome/matconvnet-calvin#12]
Hi. 

I've implemented your FRCNN code with just changing pretained model from vgg net to Alex net, and I obtained low performance (up to 49%) comparing to the existing records in some papers. Are there any tips for enhancing the performance in Alex net? e.g. learning rate , etc..

**Comment by nightrome** [comment]
Hi,

did you check that the provided VGG-16 model works as indicated? In general I am not very surprised about the 49% mAP on VOC 2007. Some things to consider are: multi-scale training, better learning parameters (especially learning rate, maybe you could use an adaptive learning rate policy like Adam) or more or different Selective Search box proposals. If you can't match the performance I would suggest using the author's (outdated) Fast R-CNN implementation in Caffe. Even if you don't want to mess with Caffe you could train and test your model there and use the predictions in Matlab.

**Comment by Changjae** [comment]
Thanks for your quick response!
As you said, I'll change the SGD solver and the # of proposals. Is there any option for multi-scale training in your code?? or I may have to code it or try the Caffe implementation

**Comment by nightrome** [comment]
Unfortunately not! And from the paper I get that the improvement should only be about 1% mAP, so I am not sure it is worth the effort.
